### ISSUE TITLE
Series x controller issue

### ISSUE THREAD

**Issue by maddogspeighty** [mistertest/devtest#2]
I have a shield tv 2017, I have my series x controller connected via Bluetooth, works fine with stadia, game pass app, geforce now and so on.

When testing your app my controller is detected in the menu when selecting a game but once I load a game my controller isn't detected and I can't go any further.

Any ideas.

**Comment by mistertest** [comment]
Hello, have you another device like keyboard or mouse connected to the shield at the same time  ?  I will test this night and come back to you.

**Comment by the-badmannn** [comment]
Sorry for writing here, but I didn't wanted to open another thread because my problem is related. With my series x controller the "back" button is registered directly by the Nvidia Shield as back and can't be used in game, this also happens with the Xbox button.

**Comment by mistertest** [comment]
> Sorry for writing here, but I didn't wanted to open another thread because my problem is related. With my series x controller the "back" button is registered directly by the Nvidia Shield as back and can't be used in game, this also happens with the Xbox button. Also not related to this, but could you please make a web app for Amazon Luna? Pretty please 🙏

hello, i created it here : https://github.com/mistertest/devtest/issues/3#issue-929563251 
For Luna, i don't have Luna, i'm french and the service is not present in Europa.

**Comment by maddogspeighty** [comment]
No, I don't have any other controller or keyboard connected.

**Comment by mistertest** [comment]
> No, I don't have any other controller or keyboard connected.

Have youu updated the "webview" app  in your playstore ?

**Comment by the-badmannn** [comment]
the website for luna is: https://www.amazon.com/luna
can you please try and make it?

**Comment by mistertest** [comment]
> the website for luna is: https://www.amazon.com/luna
> can you please try and make it?

Honestly, i'm interesting about that, but, we haven't access to the closed beta in France. So i'll need a vpn and a valide accout from united states. If you have this account, and if you can taking me your logins during the developpent i can test somethings.

**Comment by mistertest** [comment]
> No, I don't have any other controller or keyboard connected.

Hello man, thank you for the discussion about that in CGX discord. So it will be interesting to let open this issue and see.

**Comment by mistertest** [comment]
Apparently this problem is not related to the application but rather to the windows eco system seen that other hardware partners of Microsoft also have this concern.

**Comment by cyberden** [comment]
Hello, I have the same issue as OP. The series X controller works fine in the webview to select the game but once the stream starts no input is registered.

I have a "keyboard" (harmony hub) connected to the shield.

**Comment by maddogspeighty** [comment]
That's a good point, I have a harmony hub connected as well to my shield.

**Comment by mistertest** [comment]
> That's a good point, I have a harmony hub connected as well to my shield.

Yes for now i don't know why but others physical hardware no official to the shield can block the controller. I will study this issue later for futures versions.

**Comment by cyberden** [comment]
I updated my xbox controller, rebooted the shield and reconnected the controller and it worked as expected. The shield hub is still connected.

However, the back button mapped to the controller makes it unplayable but it's a nice try.

**Comment by mistertest** [comment]
> I updated my xbox controller, rebooted the shield and reconnected the controller and it worked as expected. The shield hub is still connected.
> 
> However, the back button mapped to the controller makes it unplayable but it's a nice try.

hello happy to hear that. Yes the back/select is a issue already resolved, i will publish the wiki and the new apk today. So yes you can enjoy it and play but without the select button, you can't level up your skills, using the map, it's primordial for playing a modern game, for sure. But you can enjoy Hallo5 and others games that not using the game select menu for now.

**Comment by jholloway** [comment]
I have something to add here... I have the Onn UHD Streaming Box. I installed V4 of the XcloudBeta software. My Android TV box has two devices connected - the included Onn Remote and an official Xbox One wireless controller via Bluetooth. The Xbox controller can control the interface, but once launching into a game, it doesn't work.

If I unpair the Onn Remote from the Onn box, the Xbox controller starts working in game.

Ideally I would be able to keep both devices paired and usable with the interface and default to the Xbox controller for the game. Is it possible for you to incorporate this change?

**Comment by mistertest** [comment]
> I have something to add here... I have the Onn UHD Streaming Box. I installed V4 of the XcloudBeta software. My Android TV box has two devices connected - the included Onn Remote and an official Xbox One wireless controller via Bluetooth. The Xbox controller can control the interface, but once launching into a game, it doesn't work.
> 
> If I unpair the Onn Remote from the Onn box, the Xbox controller starts working in game.
> 
> Ideally I would be able to keep both devices paired and usable with the interface and default to the Xbox controller for the game. Is it possible for you to incorporate this change?


Hello yes i will working on this and similar issues with multiple devices connected for future version.

**Comment by mistertest** [comment]
> I have something to add here... I have the Onn UHD Streaming Box. I installed V4 of the XcloudBeta software. My Android TV box has two devices connected - the included Onn Remote and an official Xbox One wireless controller via Bluetooth. The Xbox controller can control the interface, but once launching into a game, it doesn't work.
> 
> If I unpair the Onn Remote from the Onn box, the Xbox controller starts working in game.
> 
> Ideally I would be able to keep both devices paired and usable with the interface and default to the Xbox controller for the game. Is it possible for you to incorporate this change?

Hello i opened a new issue for this.

**Comment by the-badmannn** [comment]
> > the website for luna is: https://www.amazon.com/luna
> > can you please try and make it?
> 
> Honestly, i'm interesting about that, but, we haven't access to the closed beta in France. So i'll need a vpn and a valide accout from united states. If you have this account, and if you can taking me your logins during the developpent i can test somethings. 

What's your username on Cloud Gaming Xtreme discord?

**Comment by mistertest** [comment]
> > > the website for luna is: https://www.amazon.com/luna
> > > can you please try and make it?
> > 
> > 
> > Honestly, i'm interesting about that, but, we haven't access to the closed beta in France. So i'll need a vpn and a valide accout from united states. If you have this account, and if you can taking me your logins during the developpent i can test somethings.
> 
> What's your username on Cloud Gaming Xtreme discord?

Hello my username in CGX discord is : misterdeter21 and you'll see me in the "xbox-cloud-gaming" section chat .

**Comment by wags1** [comment]
Does the controller still not work with a Harmony remote (connected via BT as a Harmony keyboard) connected? I thought this was resolved but I am using an Nvidia game controller on my Shield and I can navigate the menus and select a game but once in the game the controller does not work. I would prefer not to have to unpair my Harmony remote to play games. Thanks.
